torch/_inductor/codegen/wrapper.py

```diff
@@ -49,6 +49,10 @@ def buffer_reuse_key(node: ir.Buffer):
 
 
 def is_int(s: str):
+    # Cpp code gen adds L at the end of ints
+    # Lets remove it for checking whether we have an int or not
+    if len(s) > 1 and s[-1] == "L":
+        s = s[: len(s) - 1]
```

Comment:
To be nitpicky, maybe `if s and s[-1] == "L"` instead (or at least `len(s) > 0`, as empty string will raise exception below anyway)? Also, can do `s = s[:-1]`.